rmgpy/cantherm/statmech.py

```diff
@@ -522,9 +523,7 @@ def applyEnergyCorrections(E0, modelChemistry, atoms, bonds):
     # Step 1: Reference all energies to a model chemistry-independent basis
     # by subtracting out that model chemistry's atomic energies
     # Note: If your model chemistry does not include spin orbit coupling, you should add the corrections to the energies here
-    if modelChemistry == 'CBS-QB3':
-        atomEnergies = {'H':-0.499818 + SOC['H'], 'N':-54.520543 + SOC['N'], 'O':-74.987624+ SOC['O'], 'C':-37.785385+ SOC['C'], 'P':-340.817186+ SOC['P'], 'S': -397.657360+ SOC['S']}
-    elif modelChemistry == 'G3':
```

Comment:
it seems like the CBS-QB3 code got moved down, but I can't notice a change in the values. Is there a reason that I am not seeing for the moving of the code?